src/app/tests/suites/TV_ApplicationLauncherCluster.yaml

```diff
@@ -29,34 +29,51 @@ tests:
       response:
           value: [123, 456]
 
+    # TODO: Enable once attribute struct is supported
+    # - label: "Read attribute application launcher app"
```

Comment:
Please just use "disabled" instead of commenting things out.  I thought we talked about this already?

Reply:
Uncommented them out. Updated the PR.